api/prometheus/api.go

```diff
@@ -280,8 +280,10 @@ func (qr *queryResult) UnmarshalJSON(b []byte) error {
 type QueryAPI interface {
 	// Query performs a query for the given time.
 	Query(ctx context.Context, query string, ts time.Time) (model.Value, error)
-	// Query performs a query for the given range.
+	// QueryRange performs a query for the given range.
 	QueryRange(ctx context.Context, query string, r Range) (model.Value, error)
+	// LabelValues performs a query for the values of the given label.
+	LabelValues(ctx context.Context, label string) (model.LabelValues, error)
```

Comment:
So, I always understood this interface to only include `/api/v1/query*` endpoints. With that, we'd end up with `QueryAPI`, `LabelsAPI`, `AlertmanagerAPI`, etc. interfaces.

I'd say we should either do that, or rename this interface to something like `APIv1` (or a better name). Adding all current v1 functions in a `QueryAPI` interface doesn't make sense though, as not all are about querying.